### PR TITLE
Added texlab language server for .tex and .bib to config.lua

### PR DESCRIPTION
**Pull request by JonRoo**
This minimal installation and configuration of the texlab language server works for me.

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -547,6 +547,18 @@ lspconfig.sumneko_lua = add_lsp {
...
+---# LaTeX Texlab language server
+--- __Status__: Works
+--- __Site__: https://github.com/latex-lsp/texlab
+--- __Installation__: git clone https://github.com/latex-lsp/texlab.git , then inside the texlab folder, run: cargo build --release
+--- __Note__: Rust has to be installed
+lspconfig.texlab = add_lsp {
+  name = "texlab",
+  language = "latex",
+  file_patterns = { "%.tex$", "%.bib$" },
+  command = { '/path...to...texlab/target/release/texlab' }
```
Comment (jgmdev) [inline config.lua:559]: Can you replace `'/path...to...texlab/target/release/texlab'` with just `'texlab'`? So if users add the binary location to PATH it is automatically picked up by the config without having to manually specify it.

**Review by jgmdev — CHANGES_REQUESTED** [review]

File: config.lua [hunk 2]
```diff
@@ -547,6 +547,18 @@ lspconfig.sumneko_lua = add_lsp {
...
 
+---# LaTeX Texlab language server
+--- __Status__: Works
+--- __Site__: https://github.com/latex-lsp/texlab
+--- __Installation__: git clone https://github.com/latex-lsp/texlab.git , then inside the texlab folder, run: cargo build --release
+--- __Note__: Rust has to be installed
+lspconfig.texlab = add_lsp {
+  name = "texlab",
+  language = "latex",
+  file_patterns = { "%.tex$", "%.bib$" },
```
Comment (Guldoman) [inline config.lua:558]: Would it make sense to also add the file formats specified here https://github.com/lite-xl/lite-xl-plugins/blob/3eb5ef21dcb560cf41e84f7cba37e957e5138502/plugins/language_tex.lua#L6 ?

File: config.lua (hunk 1 above)
Comment (JonRoo) [reply config.lua:559]: Of course, I will do so.

File: config.lua (hunk 2 above)
Comment (JonRoo) [reply config.lua:558]: Yes, it does make sense indeed. I will add them.

**Comment by jgmdev** [comment]
Thanks for the changes, merging.

**Pull request merged**